ext/websocket/01_websocket.js

```diff
@@ -394,13 +394,14 @@ class WebSocket extends EventTarget {
 
   async [_eventLoop]() {
     while (this[_readyState] !== CLOSED) {
-      const { kind, value } = await core.opAsync(
+      const [kind, value] = await core.opAsync(
```

Comment:
Please use the object destructuring assignment for primordials.

```suggestion
      const { 0: kind, 1: value } = await core.opAsync(
```

Reply:
done, thanks.